docs/modules/ROOT/pages/index.adoc

```diff
@@ -17,21 +17,21 @@ for tables having Change Data Capture (CDC) enabled to https://www.datastax.com/
 {cdc_cass} has two components:
```

Comment:
We should say other than a C* and a Pulsar cluster, {cdc_cass} has two components ...